### PR TITLE
Canavandl/fix bokeh element ids

### PR DESCRIPTION
**Pull request by canavandl**
fix ids

### CODE REVIEW

File: src/widget.ts [hunk 1]
```diff
@@ -18,20 +18,23 @@ class DistributedUIElement extends Widget {
...
+    // store bokeh model id as private attr for access in onResize eventing
+    this._bokeh_id = script["data-bokeh-model-id"]
+
     let tag = document.createElement('script')
     tag.src = script.src
     tag.id = script.bokeh_id
     tag.setAttribute('data-bokeh-model-id', script['data-bokeh-model-id'])
     tag.setAttribute('data-bokeh-doc-id', script['data-bokeh-doc-id'])
+    tag.onload = (event: Event) => {
+      // this._plot_ref = Bokeh.index[this._bokeh_id].model.document
```
Comment (canavandl) [inline src/widget.ts]: this needs to set an attr on the parent

**Comment by canavandl** [comment]
screen shot 2016-09-27 at 1 14 39 pm

**Comment by mrocklin** [comment]
Thoughts on vertical resizing?

**Comment by canavandl** [comment]
There was an issue where loading a new Bokeh element via script tag would override the `Bokeh.index`, so opening a second panel would cause the ref to the first plot to be lost.

The resizing now works because of a length onload callback on the script tag. Essentially, it has to wait long enough for bokehjs to load, the plot to render and be added to the global Bokeh.index, so that a reference the model can be stored as an attr on our Widget subclass.

The consequence of this is that the first plot won't resize (because the onResize event fires too quickly), but every subsequent new plot/panel drag event will make everything work as expected.

**Comment by canavandl** [comment]
> Thoughts on vertical resizing?

I think the default sizing mode to element is `scale_width` and we might just have to change it to `scale_both`. I'll investigate

**Comment by mrocklin** [comment]
I anticipate scale_both to be better for dask.distributed plots

**Comment by canavandl** [comment]
superceded by #3

**Pull request closed without merging**